pkg/api/meta/help.go

```diff
@@ -39,6 +39,14 @@ func GetItemsPtr(list runtime.Object) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// if we're a runtime.Unstructured, check to see if we have an `items` key
+	if unstructured, ok := list.(*runtime.Unstructured); ok {
+		if items, ok := unstructured.Object["items"]; ok {
+			return items, nil
```

Comment:
Doesn't this break https://github.com/kubernetes/kubernetes/pull/36085/files?diff=unified#diff-0787cfd43bbaea56d9219ec8b2d456b6L109 because `EnforcePtr` returns an error on non addressable, non-pointer arguments?